src/packages/carpool-view-flow/react/screens/trip-form/TripFormScreen.jsx

```diff
@@ -112,54 +72,49 @@ class TripForm extends React.Component {
     });
   }
 
-  closeErrorSnackbar() {
-    //d("Close snackbar")
-    this.setState({
-      errorSnackbarOpen: false,
-    });
-  }
-
-  valueChanged(valueName, e) {
-    this.setState({[valueName]: e.target.value})
-  }
-
-  // TODO is this used?
-  fromInputUpdate (val) {
-    getLocationSuggestions(val, (suggestions) => this.setState({
-      fromSuggestions: suggestions
-    }))
-    this.setState({
-      from: val,
-    })
-  }
-
-  fromInputSelect (val) {
-    this.setState({
-      from: val,
-      fromLoc: undefined // reset location received from url
-    })
-  }
-
-  toInputUpdate (val) {
-    getLocationSuggestions(val, (suggestions) => this.setState({
-      toSuggestions: suggestions
-    }))
-    this.setState({
-      to: val,
-    })
+  openDateTimePicker () {
+    this.picker.openDateTimePicker(this.state.isDepartureDate, this.props.tripDateTime)
   }
 
-  toInputSelect (val) {
-    this.setState({
-      to: val,
-    })
-  }
+  // submitForm () {
```

Comment:
Could you add line 
// TODO <some action that should be done with those comments>